client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/provisioning/index.tsx

```diff
@@ -46,7 +44,7 @@ export const Provisioning: FC< ProvisioningProps > = ( { status } ) => {
 		};
 
 		text = translate(
-			'Sorry, we couldn’t finish setting up your site. {{link}}Please, contact support{{/link}}.',
+			'Sorry, we couldn’t finish setting up your site. {{link}}Please contact support{{/link}}.',
```

Comment:
I think this link also needs the icon to show that it opens in a new tab.

Reply:
Good catch! Fixed in https://github.com/Automattic/wp-calypso/pull/92608/commits/1d1defd71453693a3f08ac83aaf4234973e2fb74